scripts/upgrade-blueprint-2.0.0-rename

```diff
@@ -99,42 +94,103 @@ function rename() {
         --exclude-dir=$EXCLUDE_GLOB \
         . \
         ${SEARCH_PATH} \
-        | xargs perl -lne 'if (/$findRegexp/) { print \$ARGV; close ARGV }'
+        | xargs perl -lne 'if (/$1/) { print \$ARGV; close ARGV }'
     "
-    files=$(eval $cmd)
+    echo "$(eval $cmd)"
+}
+
+function rename() {
+    # Name parameters
+    fromString=$1
+    toString=$2
+    findRegexp=$3
+    files=$(matchingFiles "$findRegexp")
 
     if [[ -z ${files// } ]]; then
-        echo -e "No files contain ${RED}${fromString}${RESET}"
+        echo -e "No files contain ${BLUE}${fromString}${RESET}"
     else
         count=$(echo "$files" | wc -l | awk '{print $1}')
-        echo -e "Renaming ${RED}${fromString}${RESET} -> ${GREEN}${toString}${RESET} in ${BLUE}${count}${RESET} files"
+        echo -e "Renaming ${BLUE}${fromString}${RESET} -> ${GREEN}${toString}${RESET} in ${BLUE}${count}${RESET} files"
 
         # Iterate and search&replace
         echo "$files" | while read -r file ; do
             echo -n "$file ... "
 
             # Search&replace in place with perl
-            perl -p -i -e "s/$findRegexp/$toString/g" $file
+            perl -p -i -e "s/$findRegexp/$toString/g" "$file"
             echo -e "${GREEN}done${RESET}"
         done
     fi
 }
 
-# Renames
-
-### Table
-rename allowMultipleSelection enableMultipleSelection
-rename fillBodyWithGhostCells enableGhostCells
-rename isColumnReorderable enableColumnReordering
-rename isColumnResizable enableColumnResizing
-rename isRowHeaderShown enableRowHeader
-rename isRowReorderable enableRowReordering
-rename isRowResizable enableRowResizing
-rename enableFocus enableFocusedCell
-rename onFocus onFocusedCell
-rename renderBody bodyRenderer
-rename renderBodyContextMenu bodyContextMenuRenderer
-rename renderCell cellRenderer
-rename renderColumnHeader columnHeaderCellRenderer
-rename renderRowHeader rowHeaderCellRenderer
-rename useInteractionBar enableColumnInteractionBar
+function warn() {
+    # Name parameters
+    fromString=$1
+    toString=$2
+    findRegexp=$3
+    files=$(matchingFiles "$findRegexp")
+
+    if [[ -z ${files// } ]]; then
+        echo -e "No files contain ${BLUE}${fromString}${RESET}"
+    else
+        count=$(echo "$files" | wc -l | awk '{print $1}')
+        echo ""
+        echo -e "${RED}WARNING: Skipping ${BLUE}${fromString}${RESET} -> ${GREEN}${toString}${RESET}"
+        echo -e "${RED}Since ${BLUE}${fromString}${RED} is very common, you'll need to manually rename this string.${RESET}"
+        echo -e "${RED}It was found in these files ${BLUE}${count}${RED} files:${RESET}"
+        echo "$files"
+    fi
+}
+
+function renameProp() {
+    # Add prefix and word boundaries to search string
+    rename $1 $2 "$PREFIX\\b$1\\b"
+}
+
+function warnProp() {
+    warn $1 $2 "$PREFIX\\b$1\\b"
+}
+
+function renamePartialClass() {
+    # Don't add word boundary so that partial css classnames rename
```

Comment:
invalid comment?

Reply:
nope, this is still relevant, notice how we don't add the the `\\b` to this one.